subprojects/launcher/src/test/groovy/org/gradle/launcher/cli/converter/PropertiesToDaemonParametersConverterTest.groovy

```diff
@@ -137,18 +137,4 @@ class PropertiesToDaemonParametersConverterTest extends Specification {
         true    | true
         false   | false
     }
-
-    def "enable debug mode from JVM args when default debug argument is used"() {
```

Comment:
❓ Same feeling about losing some of the test coverage.

Reply:
Same as https://github.com/gradle/gradle/pull/9863#discussion_r306775088